CantoboardFramework/Keyboard/View/Status/StatusMenu.swift

```diff
@@ -83,15 +80,18 @@ class StatusMenu: UIView {
         super.layoutSubviews()
         layer.shadowPath = CGPath(roundedRect: bounds, cornerWidth: Self.cornerRadius, cornerHeight: Self.cornerRadius, transform: nil)
         
+        let rowHeight = bounds.height / CGFloat(itemLabelInRows.count)
+        let font = UIFont.systemFont(ofSize: bounds.width * CGFloat(itemLabelInRows.count) * Self.fontSizeRatio) // intentional
```

Comment:
If I'm reading the code correctly, are you changing the font size calculation from height based to width based?

Reply:
Ah, OK. Now it should be mathematically the same while improving readability.